apps/preview/nuxt/pages/showcases/MegaMenu/BaseMegaMenu.vue

```diff
@@ -135,11 +137,14 @@ import watch from '@assets/watch.png';
 
 const { isOpen, toggle, close } = useDisclosure();
 const menuRef = ref();
+const drawerRef = ref();
 
 onClickOutside(menuRef, () => {
   close();
 });
 
+useTrapFocus(drawerRef, { activeState: isOpen, arrowKeysOn: true, arrowFocusGroupSelector: '.list-item' });
```

Comment:
do we need this group? `a` which is inside is focusable so it should working without `.list-item` group

Reply:
@Szymon-dziewonski removed :)